lib/dom/dom.nit

```diff
@@ -67,3 +67,27 @@ redef class XMLStartTag
 		abort
 	end
 end
+
+redef class XMLAttrTag
+
+	# Attributes as a map (ignoring malformed attributes)
```

Comment:
Agreed, this job should be left to the parser, I'd suggest adding a strict mode to it if there is a need for that, but that's outside the scope of this PR.
Maybe open an issue for that if such a behaviour has not already been implemented?